nuxt.config.ts

```diff
@@ -37,6 +37,7 @@ export default defineNuxtConfig({
         '~/components/atoms',
         '~/components/molecules',
         '~/components/organisms',
+        '~/components/pages',
```

Comment:
IMO this could lead to misunderstandings with the root `pages/` folder.

Reply:
Removed